testing/admin_key.txt

```diff
@@ -0,0 +1 @@
+0135d8598650f8f5cb0f30c34ec2e2bb62793bc28717c8eb6fb577996d50be5f4281b59181095065c5d0f86a2c31ddbe9b597ec62b47ded69782cd
```

Comment:
snip?

Reply:
What do you mean by this? We need this admin key (it's the same one used for all local backends). Sticking it in a file was slightly better than copying it to multiple places